pkg/clusterapi/extraargs.go

```diff
@@ -49,6 +49,15 @@ func PodIAMAuthExtraArgs(podIAMConfig *v1alpha1.PodIAMConfig) ExtraArgs {
 	return args
 }
 
+func ResolvConfExtraArgs(resolvConf v1alpha1.ResolvConf) ExtraArgs {
+	if resolvConf.Path == "" {
```

Comment:
We can actually remove this if condition if we make use of the `AddIfNotEmpty` function